### PR TITLE
CSharpStyle wrapper PoC for one method

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes # .
Added CSharp wrapper PoC

**Changes proposed in this request**
Differences between Android style API and CSharp style API (using wrapper)
How to convert callback to async/awai

**Testing**
Manual

**Performance impact**
Demo. None

### CODE REVIEW

File: samples/Bogdan.TestApp/AndroidWrapper.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            // create instance of the callback
+            var interactiveCallback = new InteractiveAuthCallback(
+                onCancelAction: () => {
+                    // on cancel, set results to canceled
+                    tcsResult.SetCanceled();
+                },
+                onErrorAction: (ex) =>
+                {
+                    // throw any excpetion
+                    throw ex;
```
Comment (bgavrilMS) [inline samples/Bogdan.TestApp/AndroidWrapper.cs:67]: What is the difference between this and `tcsResult.SetException(ex)` ? Which one simulates how the `await` works best?

Comment (SameerK-MSFT) [reply samples/Bogdan.TestApp/AndroidWrapper.cs:67]: Good catch. Will fix.

**Review by trwalke — APPROVED** [review]
Looks like a great set of first steps. Thanks Sameer

**Comment by bgavrilMS** [comment]
Closing as it was for demo purposes only.

**Pull request closed without merging**